src/Dashboard/Createproperty.jsx

```diff
@@ -44,16 +44,29 @@ console.log(config, getToken)
   };
 
   console.log(formData)
+  const check = process.env.REACT_APP_API_URL
+  console.log(`${process.env.REACT_APP_API_URL}/agent/properties`)
 
   const handleSubmit = async (e) => {
     e.preventDefault();
     try {
-      const response = await axios.post(url, { ...formData }, config);
+      const response = await axios.post(`${process.env.REACT_APP_API_URL}/agent/properties`, { ...formData }, config);
```

Comment:
Since you already save process.env... into a variable, you don't need to write it all over again, you can just use the 'check' instead.